.eslintrc.js

```diff
@@ -24,11 +24,17 @@
       "argsIgnorePattern": "(^reject$|^_$)",
       "varsIgnorePattern": "(^_$)"
     }],
+    "brace-style": [2, "1tbs", {
+      "allowSingleLine": true
```

Comment:
aye. i was going for `if (foo) bar();` which is handled under `curly` instead. And the style we agreed on last call isn't an option: http://eslint.org/docs/rules/curly

so i'll revert this bit.